### PR TITLE
refactor SyslogFrameProcessor a bit

### PR DESCRIPTION
**Pull request by kortemik**

### CODE REVIEW

**Review by eemhu — APPROVED** [review]
looks good to me. Perhaps the string comparison (relpFrame().command().toString()) could be replaced with a decorator pattern: "new RelpCommand(relpFrame().command())" to output RelpCommand enum instead of string.

File: src/main/java/com/teragrep/rlp_03/SyslogFrameProcessor.java [hunk 1]
```diff
@@ -65,70 +67,34 @@ public class SyslogFrameProcessor implements FrameProcessor, AutoCloseable {
...
-                } else {
-                    txFrameList.add(createResponse(frameContext.relpFrame(), RelpCommand.RESPONSE, "500 NO PAYLOAD"));
-
-                }
-                break;
+       String relpCommand = frameContext.relpFrame().command().toString();
 
-            default:
-                break;
+       Consumer<FrameContext> commandConsumer = relpCommandConsumerMap.get(relpCommand);
```
Comment (StrongestNumber9) [inline src/main/java/com/teragrep/rlp_03/SyslogFrameProcessor.java:91]: Using `relpCommandConsumerMap.getOrDefault(relpCommand, new RelpEventServerClose())` should be fine?

**Comment by StrongestNumber9** [comment]
> looks good to me. Perhaps the string comparison (relpFrame().command().toString()) could be replaced with a decorator pattern: "new RelpCommand(relpFrame().command())" to output RelpCommand enum instead of string.

I agree, no need for toString() here in my opinion

**Comment by kortemik** [comment]
https://github.com/teragrep/rlp_01/blob/master/src/main/java/com/teragrep/rlp_01/RelpCommand.java this is not an enum, and I think we must make the server so that the commands can be added by user of the server library by providing a handler for them

**Pull request merged**